src/lava/proc/plateau/models.py

```diff
@@ -0,0 +1,130 @@
+# Copyright (C) 2023 Intel Corporation
```

Comment:
@tim-shea , is code copyright by Intel if written by an external researcher?

Reply:
This is fine. Someone can give copyright to whomever they wish. We direct people to add copyright like this:
https://lava-nc.org/developer_guide.html#add-a-license